### ISSUE TITLE
`fixBigInt` transforms all `GET` request `res.body`s to string. 

### ISSUE THREAD

**Issue by iritmaximus** [nowcommunity/nowdatabase#293]
https://github.com/nowcommunity/nowdatabase/blob/64fe98f7488bc7e084d235d3c7fc7ffa354a3fb1/backend/src/utils/common.ts#L3-L5 It seems there is no particular need to implement `fixBigInt` this way. 

In fact, few `service`s that utilize the `fixBigInt` parse the returned `string` back to `object`. 
https://github.com/nowcommunity/nowdatabase/blob/64fe98f7488bc7e084d235d3c7fc7ffa354a3fb1/backend/src/services/species.ts#L70

**Comment by iritmaximus** [comment]
Also this transformation is made for only some paths: 
https://github.com/nowcommunity/nowdatabase/blob/64fe98f7488bc7e084d235d3c7fc7ffa354a3fb1/backend/src/routes/species.ts#L13
but not
https://github.com/nowcommunity/nowdatabase/blob/64fe98f7488bc7e084d235d3c7fc7ffa354a3fb1/backend/src/routes/species.ts#L20

**Comment by ShootingStar91** [comment]
Indeed, best to do it in one level, I probably meant to refactor it into services but forgot some of it.

**Comment by karilint** [comment]
Looks resolved on main: `fixBigInt` no longer returns a JSON string. It now returns a JSON-safe object (BigInt -> Number), so Express responses using `res.send(fixBigInt(...))` have `application/json` and Supertest `res.body` is no longer a string.

Fix introduced in commit `05e707e4` ("Fix JSON content-type for API responses").

Proposing to close this issue unless there’s still an endpoint returning a string response body.
